packages/opencensus-propagation-b3/src/b3-format.ts

```diff
@@ -59,14 +47,13 @@ export class B3Format implements Propagation {
    * @param spanContext
    */
   inject(setter: HeaderSetter, spanContext: SpanContext): void {
-    if (setter) {
-      setter.setHeader(X_B3_TRACE_ID, spanContext.traceId || 'undefined');
-      setter.setHeader(X_B3_SPAN_ID, spanContext.spanId || 'undefined');
-      if (spanContext && (spanContext.options & SAMPLED_VALUE) !== 0) {
-        setter.setHeader(X_B3_SAMPLED, `${SAMPLED_VALUE}`);
-      } else {
-        setter.setHeader(X_B3_SAMPLED, `${NOT_SAMPLED_VALUE}`);
-      }
+    setter.setHeader(X_B3_TRACE_ID, spanContext.traceId || '');
```

Comment:
On a similar note as above, I'm not sure if we should be writing an empty value to a header. That being said the [`SpanContext` interface](https://github.com/census-instrumentation/opencensus-node/blob/3111ab8c29b5d60e68a762c439ef909e3af9589b/packages/opencensus-core/src/trace/model/types.ts#L252-L261) should guarantee that `traceId` and `spanId` are not blank.

Reply:
Agreed, does not make sense to write empty values to a header. Added check for the same. PTAL